docs/foundational_concepts.md

```diff
@@ -86,11 +86,21 @@ For example, here's an excerpt of the Wolfi secdb:
 
 From this data, we can see that the Wolfi package named `apko`, in its version `0.7.3-r1`, resolved three CVEs: `CVE-2023-28840`, `CVE-2023-28841`, and `CVE-2023-28842`.
 
-For more information on how to apply this data to a vulnerability scan, see [Scanning Implementation](./scanning_implementation.md).
+### OSV feed (experimental)
```

Comment:
i would change to `beta` :)

Reply:
Yeah, I'm going to ditch "experimental". I think I'm going to just say that it's new, and we can remove that "new" status in several months' time.